src/main/java/org/apache/maven/shared/jar/JarData.java

```diff
@@ -87,13 +98,19 @@ public JarData(File file, Manifest manifest, List<JarEntry> entries) {
         this.entries = Collections.unmodifiableList(entries);
 
         boolean aSealed = false;
+        boolean multiRelease = false;
         if (this.manifest != null) {
             String sval = this.manifest.getMainAttributes().getValue(Attributes.Name.SEALED);
             if (sval != null && !sval.isEmpty()) {
                 aSealed = "true".equalsIgnoreCase(sval.trim());
             }
+            String sMultiRelease = this.manifest.getMainAttributes().getValue(new Attributes.Name("Multi-Release"));
```

Comment:
I'd reuse `sval` here

Reply:
seems clearer to extract a method to handle both sealed and multirelease assignments. WDYT?

Reply:
Yes, I do agree.